tests/Feature/Api/UserTaste/TasteTest.php

```diff
@@ -33,7 +33,9 @@ public function test_get_tastes()
     {
         factory(Taste::class, 3)->create([
             'user_id' => $this->user->id
```

Comment:
'user_id' => $this->user->id - can be removed?

Reply:
we need to assign custom taste to current user, so i think no